### PR TITLE
add project detail view and delete functionality

### PR DESCRIPTION
**Pull request by ashleygwilliams**
this is the single project view and delete funcitonality.

note, delete does not work on the server yet. you can delete on client til all models are gone and then refresh the page (which pulls from the server) to see that they are all still there.

### CODE REVIEW

**Comment by jbuck** [comment]
@ashleygwilliams I'm not able to click the featured checkbox or the delete button? Like it clicks, but it doesn't do anything:

https://www.dropbox.com/s/1ulguvfwebf56mj/Screenshot%202015-08-11%2012.51.33.png?dl=0

**Comment by ashleygwilliams** [comment]
oh yeah i remvoed those buttons but didnt add them to this PR since they weren't part of the feature

**Comment by ashleygwilliams** [comment]
delete works from the detail

File: app/adapters/application.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import DS from 'ember-data';
 
 export default DS.RESTAdapter.extend({
-    host: 'https://api.webmaker.org'
+    host: 'http://localhost:2015'
```
Comment (jbuck) [inline app/adapters/application.js:4]: Out of sheer curiosity, does ember support some way of injecting config into the build process?

**Comment by jbuck** [comment]
@ashleygwilliams r+!

**Pull request merged**